tests/lit-tests/2468.ispc

```diff
@@ -0,0 +1,23 @@
+// RUN: %{ispc} %s --target=avx2-i32x8 --arch=x86-64 --nostdlib --emit-asm -o - | FileCheck %s
+
+// REQUIRES: X86_ENABLED
+
+// The goal of this test is to check that code generation results into a pair of loads, an add, and a store
```

Comment:
It's a pair of loads, technically, but only one `vmovups` is expected - because the second one is embedded in `vaddps` instruction. Would be good to mention it, otherwise it's not clear why one load is expected in the checks.